wfe/wfe.go

```diff
@@ -1100,7 +1100,10 @@ func (wfe *WebFrontEndImpl) Registration(ctx context.Context, logEvent *requestE
 		return
 	}
 
-	if len(update.Agreement) > 0 && update.Agreement != wfe.SubscriberAgreementURL {
+	// Only fail if the sent URL doesn't match the currently valid agreement
+	// and that it doesn't match the URL currently in the registration
```

Comment:
The "and that" here isn't right. Also, this comment explains _what_ happens, but it would be more useful to explain _why_ (a good general rule of thumb for comments).